tests/bootstrap.php

```diff
@@ -91,6 +91,14 @@ function _manually_load_plugin() {
 	update_option( 'woocommerce_calc_taxes', 'yes' );
 	define( 'WOOCOMMERCE_ADMIN_ONBOARDING_ENABLED', true );
 
+	// Fix alpha builds of WP (like on VVV).
+	global $wp_version;
+	$alpha_tag = strpos( $wp_version, '-alpha' );
+	if ( false !== $alpha_tag ) {
+		// phpcs:ignore
```

Comment:
Can we change this and line #44 to `// phpcs:ignore WordPress.WP.GlobalVariablesOverride` so that it passes the sniffs without warning?